pipelines/deployer/deployer.yaml

```diff
@@ -790,3 +842,10 @@ jobs:
       action: apply
     get_params:
       action: apply
+  - put: namespace-role-state
+    params:
+      env_name: ((account-name))
+      terraform_source: namespace-roles-terraform
+      action: apply
```

Comment:
pretty sure `apply` is the default and this is not required... same goes for get_params

Reply:
This is copied from user-state above, although shouldn't that be action: destroy here?

Reply:
Let's come back to user-state in another PR, I think this should be destroy.

Reply:
Opened #323